test/e2e/crcsuite/crcsuite.go

```diff
@@ -161,6 +159,18 @@ func FeatureContext(s *godog.Suite) {
 	})
 }
 
+func defaultBundleForOs(bundleVersion string) map[string]string {
+	return map[string]string{
+		"darwin":  fmt.Sprintf("crc_hyperkit_%s.crcbundle", bundleVersion),
+		"linux":   fmt.Sprintf("crc_libvirt_%s.crcbundle", bundleVersion),
+		"windows": fmt.Sprintf("crc_hyperv_%s.crcbundle", bundleVersion),
+	}
+}
+
+func getBundleFosOs(os, bundleVersion string) string {
```

Comment:
Can you fix the `FosOs` Type while you are moving this? The intermediate helper is no longer needed as it only has one user.